src/Api/Security/AdminAuthenticationProvider.php

```diff
@@ -110,12 +110,13 @@ public function clearCache(): void
 
     private function getMboToken(): string
     {
-        $employee = $this->context->employee;
-        if (!($employee instanceof \Employee)) {
-            throw new UnauthorizedException('User is not connected');
+        if($this->context && $this->context->employee && $this->context->employee instanceof \Employee) {
```

Comment:
```suggestion
        if($this->context->employee instanceof \Employee) {
```
I believe you can simplify this way to make PHPStan happier